### PR TITLE
Karan vesting

### PR DESCRIPTION
**Pull request by darthsiroftardis**
Addressed changes that were requested

### CODE REVIEW

File: contract/Cargo.toml [hunk 1]
```diff
@@ -7,15 +7,10 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-casperlabs-contract = "0.4.1"
-casperlabs-types = "0.4.1"
+casperlabs-contract = { git="https://github.com/CasperLabs/CasperLabs", branch="hackathon" }
```
Comment (zie1ony) [inline contract/Cargo.toml:10]: Change to the same in `test` crate.

File: contract/Cargo.toml [hunk 2]
```diff
@@ -7,15 +7,10 @@ edition = "2018"
...
 logic = { path = "../logic", package = "logic" }
 
-#[lib]
-#crate-type = ["cdylib"]
-#bench = false
-#doctest = false
-#test = false
 
 [features]
 default = ["casperlabs-contract/std", "casperlabs-types/std", "casperlabs-contract/test-support"]
```
Comment (zie1ony) [inline contract/Cargo.toml:16]: We don't need `test-support` feature in `contract` crate.

File: contract/Cargo.toml [hunk 3]
```diff
@@ -7,15 +7,10 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```
Comment (zie1ony) [inline contract/Cargo.toml:7]: I have noticed that `tests/wasm/contract.wasm` is included into the repository. Please remove it and add it to `.gitignore`.

Comment (darthsiroftardis) [reply contract/Cargo.toml:7]: I added it to .gitignore

Comment (darthsiroftardis) [reply contract/Cargo.toml:7]: Sorry, misread it as `target` instead of `target`

**Pull request merged**